controller.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class ObjController {
+
+  function ObjController() {} //constructor
+
+  function valid($x, $y, $f) { //Validate inputs.
+    if ((empty($x) && is_numeric($x) == false) || (empty($y) && is_numeric($y) == false) || (empty($f))) { //At least one textbox is empty.
```

Comment:
From what I understand you just need to check if $x is numeric, if it's empty is_numeric should return false nonetheless.